model/nc_news-model.js

```diff
@@ -11,4 +12,23 @@ exports.fetchTopics = () => {
         return err.code
     }
 })
-}
\ No newline at end of file
+}
+
+exports.selectArticleByID = (article_id) => {
+    return db.query('SELECT * FROM articles WHERE article_id = $1;', [article_id])
+    .then((result) => {
+        if (result.rows.length === 0) {
+            return Promise.reject({status: 404, msg: `No article found under ID: ${article_id}`})
+        }
+        return result.rows[0]
+    })
+    //.catch((err) => {
```

Comment:
Great use of $ syntax to protect from SQL injection 
Please remove commented out code from finalised code 🗑️